### DISCUSSION TITLE
绳网委托 5000丁尼

### DISCUSSION THREAD

**Discussion by Tartaglia770** [share121/inter-knot#3268] [其他]
谁知道绳网怎么回复吗<img src="https://bbs-static.miyoushe.com/static/2024/08/07/e6ca84d618546c1b2547ed668c1a3d01_5567412182439765172.png" alt="杜林-惊喜" width="75" >

**Comment by lllliufeng** [comment]
接了
Add a comment

**Comment by wyqsagiri** [comment]
这个委托我接了！

**Comment by Zou-pig** [comment]
诶呀来晚了

**Reply by Tartaglia770** [reply]
开心我会回复了
<img src="https://bbs-static.miyoushe.com/static/2024/06/12/580c5fe822ac37260b8c95decbd5bde6_4418994548529724081.png" alt="希格雯-护肤" width="75" >

**Comment by bbbbochi** [comment]
回复

**Comment by cleo66** [comment]
这个委托我包了@<img src="https://img-static.mihoyo.com/communityweb/upload/d2ebde3faa8c113f5c54b171d907f271.png" alt="比利 摇摆" width="75" >

**Comment by Huan-m** [comment]
为啥回复要重开一个页面啊

**Comment by kdhfkc** [comment]
我不知道
